lib/seek/extended_metadata_type/extended_metadata_type_schema.json

```diff
@@ -0,0 +1,120 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "title": {
+      "type": "string"
+    },
+    "supported_type": {
+      "type": "string",
+      "enum": [
+        "Investigation",
+        "Study",
+        "Assay",
+        "Collection",
+        "DataFile",
+        "Document",
+        "Event",
+        "Model",
+        "Presentation",
+        "Sop",
+        "Project",
+        "ExtendedMetadata"
```

Comment:
I don't these should go in the schema, but instead check programmatically that the type exists and `supports_extended_metadata?`

Reply:
I believe the schema should include the supported types. This not only serves as validation for the allowed types but also helps users when creating their JSON files. 
Would checking the supported types programmatically create duplication if they're already defined in the schema?

Reply:
I have added a validation in [ExtendedMetadataType model ](https://github.com/seek4science/seek/blob/0c653980d2484513d87a701946900340c8b0efd5/app/models/extended_metadata_type.rb#L10) , but i think we should not remove them from schema.